README.md

```diff
@@ -6,6 +6,7 @@ The observability bundle provides all necessary components to enable observabili
 
 ## Apps
 
+* alloy-app
```

Comment:
```suggestion
* alloy-logs
```
?

Reply:
AFAIK those are the name of the app as in gitrepository name and not internal names.

Reply:
I think we just went that way because the apps are named the same but whatever :D